Newsletters: add 155 (2021-06-30) (#597)
bitcoinops/bitcoinops.github.io: en/tools/calc-size.md

  input.  Used by [BIP68][] and [BIP125][], with other values having no
  defined meaning

> - **Witness item count** ([compactSize][]/4) The number of witness
>   elements included in transactions containing at least one segwit
>   input. 0.25 vbytes for up to 252 elements

Reviewer: Feels like there is something missing between elements and included. Maybe:

```suggestion
- **Witness item count** ([compactSize][]/4) The number of witness
  elements. This field is included for each input if the transaction contains at least one segwit
  input. 0.25 vbytes for up to 252 elements
```

> - **Witness items** (varies) The source of witness data for in segwit

Reviewer: ```suggestion
- **Witness items** (varies) The source of witness data in segwit
```

  // Calculate the transaction's overhead (the size independent of inputs and outputs)
  if (types[type].segwit == true) {

>     witness_flag = {{page.size.marker_and_flag}};  // segwit marker, segwit flags

Reviewer: ```suggestion
    witness_flag = {{page.size.marker_and_flag}};  // segwit marker, segwit flag
```
